test/unit/common/Reducers.test.js

```diff
@@ -668,6 +668,16 @@ describe("Reducers", () => {
       const state = DiscoveryStream(undefined, {type: at.DISCOVERY_STREAM_CONFIG_CHANGE, data: {enabled: true}});
       assert.deepEqual(state.config, {enabled: true});
     });
+    it("should load feeds with DISCOVERY_STREAM_FEEDS_UPDATE", () => {
+      const data = {
```

Comment:
Please split this into setup/execute/verify blocks separated by whitespace for readability.